### ISSUE TITLE
IAsyncEnumerable issues with .net core 3.0

### ISSUE THREAD

**Issue by vadimi** [okta/okta-sdk-dotnet#348]
Hello, I'm trying to use Okta Sdk with .net core 3.0 and I also have my custom IAsyncEnumerables in the code. Unfortunately it doesn't compile as Okta Sdk uses System.Interactive.Async version 3.1.1, but .net core 3.0 already includes IAsyncEnumberable into bcl with different signature - it's a known breaking change. Here is a quick example:

```csharp
    class Program
    {
        static void Main(string[] args)
        {
            var client = new OktaClient(new OktaClientConfiguration
            {
                OktaDomain = "https://{{yourOktaDomain}}",
                Token = "{{yourApiToken}}"
            });

            client.Policies.ListPolicies("").ToList();
        }
        
        public class MyAsync : IAsyncEnumerable<string>
        {
            public ValueTask<bool> MoveNextAsync() => default;
            public ValueTask DisposeAsync() => default;
        }
    }
```

This code fails to compile with the following error:

`Program.cs(24,32): error CS0433: The type 'IAsyncEnumerable<T>' exists in both 'System.Interactive.Async, Version=3.0.3000.0, Culture=neutral, PublicKeyToken=94bc3704cddfc263' and 'System.Runtime, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'`

Any chance to add IAsyncEnumerable support from .net core 3.0/netstandard2.1? It's possible to multitarget Okta.Sdk library for example.

**Comment by laura-rodriguez** [comment]
Hi @vadimi ,

This is definitely something we want to support, but I'll have to discuss the priority of this task with the team. Stay tuned here 👍

**Comment by djlawhead** [comment]
Does this mean that `Okta.Sdk` cannot be used in projects targeting `netstandard2.1`?

**Comment by laura-rodriguez** [comment]
Hi @djlawhead, 

This is a .NET Standard 1.3 library, and it is using some dependencies that need to be updated to be fully compatible with 2.1. I haven't tried to use this with a .NET Standard 2.1 library yet to see what errors you can face, but this is definitely something we need to look into soon.

**Comment by nivosalti** [comment]
Hi,
This is an issue for us too - is there an estimation when it is fixed?

Thanks

**Comment by laura-rodriguez** [comment]
Hi @nivosalti ,
This is in our backlog already, and we are planning to look into it soon. We don't have a specific release date to share right now, but we expect to have it available as soon as we can.

**Comment by laura-rodriguez** [comment]
Fixed by #358. This fix is available in 2.0.0.
